src/main/scala/uncore/util/BlackBoxRegs.scala

```diff
@@ -22,29 +26,26 @@ import cde.{Parameters}
   *  @param q Data Output
   *  @param clk Clock Input
   *  @param rst Reset Input
+  *  @param en Write Enable Input
   *  
-  *  @param init Value to write at Reset. 
-  *  This is a constant, 
-  *  but this construction
-  *  will likely make backend flows
-  *  and lint tools unhappy.
-  * 
   */
 
-class AsyncResetReg  extends BlackBox {
+abstract class AbstractBBReg extends BlackBox {
 
```

Comment:
rocket-chip codebase doesn't put an extra line break after the opening curly brace.